Documentation/Configuration/Typo3ConfVars/SYS.rst

```diff
@@ -860,6 +860,9 @@ security.backend.enforceContentSecurityPolicy
     If enabled, the :ref:`Content Security Policy <content-security-policy>`
     is applied in backend scope.
 
+    ..  note::
+        With TYPO3 v13.0 this feature setting is always enabled.
```

Comment:
So if it is always set then this setting would be useless and we can remove it?

Reply:
Yup, in v13. In v12 you can opt-in or opt-out. This note is in v12 to inform the integrator that they cannot opt-out forever and should take care better earlier that later.